tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/mobile@2021_02_12/node_exception_weird.pysnap

```diff
@@ -135,7 +135,7 @@ app-depth-4:
 --------------------------------------------------------------------------
 app-depth-5:
   hash: "d50c59998b650da06b01b9f2b8ecc2f7"
-  tree_label: "eventFromException | captureException | invokeClient | captureException | <anonymous>"
+  tree_label: "eventFromException | captureException | invokeClient | captureException"
```

Comment:
I guess it's good that this noise is removed, but what change caused its disappearance?